bin/bootstrap.sh

```diff
@@ -19,6 +19,8 @@ git commit -m 'Initial commit' -m 'Initialized from lisk-template: https://githu
 
 # Customize project using provided project name
 git grep -l lisk-template | xargs sed -i '' "s/lisk-template/$projectname/g"
+sed -i '.bak' $'s/\t"version": .*/\t"version": "0.0.0",/' package.json
+rm package.json.bak
```

Comment:
so we are going to use the backup and remove immediately?
I think the original one looks fine tho

Reply:
Mac and everyone else have inconsistent approaches here: Mac requires and empty string, and everyone else requires that it's not an empty string.